packages/ocs/storage-pool/body.tsx

```diff
@@ -83,6 +87,12 @@ export const StoragePoolStatus: React.FC<StoragePoolStatusProps> = ({
   );
 };
 
+export const isInfrastructureInTwoNodeOneArbiterTopology = (
```

Comment:
I would simplify the name:
```suggestion
export const isTwoNodePlusArbiterTopology = (
```

Reply:
Done